processor/src/test/java/org/mapstruct/ap/test/selection/qualifier/QualifierTest.java

```diff
@@ -96,14 +96,13 @@ public void shouldMatchClassAndMethod() {
         ErroneousMapper.class
     } )
     @ExpectedCompilationOutcome(
-             value = CompilationResult.FAILED,
-            diagnostics = {
-                @Diagnostic( type = ErroneousMapper.class,
-                        kind = Kind.ERROR,
-                        line = 28,
-                        messageRegExp = "Ambiguous mapping methods found for mapping property "
-                                + "\"java.lang.String title\" to java.lang.String.*" )
-            }
+        value = CompilationResult.FAILED,
+        diagnostics = {
+            @Diagnostic(type = ErroneousMapper.class,
+                kind = Kind.ERROR,
+                line = 28,
+                messageRegExp = "Can't map property \"java.lang.String title\" to \"java.lang.String title\".*")
```

Comment:
What is the full error message here? Asking since I think that this error message will be confusing to a user.

Reply:
I'll add some more detail to the message so its more obvious.

Reply:
Thanks